src/modules/site-v2/templates/primary/map.html

```diff
@@ -0,0 +1,240 @@
+  const elegans = [];
+  const briggsae = [];
+  const tropicalis = [];
+
+
+function naturalCompare(a, b) {
+    const ax = [], bx = [];
+  a = a.strain
+  b = b.strain
+
+  a.replace(/(\d+)|(\D+)/g, function(_, $1, $2) { ax.push([$1 || Infinity, $2 || ""]) });
+  b.replace(/(\d+)|(\D+)/g, function(_, $1, $2) { bx.push([$1 || Infinity, $2 || ""]) });
+  
+  while(ax.length && bx.length) {
+    const an = ax.shift();
+    const bn = bx.shift();
+    const nn = (an[0] - bn[0]) || an[1].localeCompare(bn[1]);
+      if(nn) return nn;
+  }
+
+  return ax.length - bx.length;
+}
+
+function set_marker_color (species) {
+  if (species === 'Caenorhabditis elegans') {
+    return icon_elegans;
+  }
+  if (species === 'Caenorhabditis briggsae') {
+    return icon_briggsae;
+  }
+  if (species === 'Caenorhabditis tropicalis') {
+    return icon_tropicalis;
+  }
+}
+
+const data = {{ strain_listing|tojson|safe }}
+  data.sort(naturalCompare)
+  strain_info = data;
+  data.forEach(function(d) {
+  if (d.latitude) {
+    strain_names.push(d.strain);
+    const popup_content = `
+    <div>
+      <ul class='popup'>
+        ${d.species ? `<li><strong>Species:</strong> ${d.species}</li>` : ''}
+        ${d.isotype ? `<li><strong>Isotype:</strong> ${d.isotype}</li>` : ''}
+        ${d.isotype_ref_strain ? `<li><strong>Reference Strain:</strong> ${d.isotype_ref_strain}</li>` : ''}
+        ${d.strain ? `<li><strong>Strains:</strong> ${d.strain}</li>` : ''}
+        ${d.previous_names ? `<li><strong>Alternative Names:</strong> ${d.previous_names}</li>` : ''}
+        ${d.release ? `<li><strong>Release Date:</strong> ${String(d.release).replace(/(\d{4})(\d{2})(\d{2})/, "$1-$2-$3")}</li>` : ''}
+        ${d.release ? `<li><strong>Substrate:</strong> ${d.substrate}</li>` : ''}
+        ${d.substrate_temp ? `<li><strong>Substrate Temp (C):</strong> ${d.substrate_temp}</li>` : ''}
+        ${d.ambient_humidity ? `<li><strong>Ambient Humidity:</strong> ${d.ambient_humidity} </li>` : ''}
+        ${d.sampled_by ? `<li><strong>Sampled by:</strong> ${d.sampled_by}</li>` : ''}
+        <li><strong>Peel-1 Zeel-1 inc:</strong> </li>
+      </ul>
+    </div>`
+
+    const m = L.marker([d.latitude, d.longitude], { icon: set_marker_color(d.species),
+                                                    species: d.species})
+                                                    .on('mouseover', function(){
+                                                      m.setIcon(icon_hover)
+                                                    })
+                                                    .on('mouseout', function(){
+                                                      const original_icon = set_marker_color(d.species)
+                                                      m.setIcon(original_icon)
+                                                    })
+                            
+    m.bindPopup(popup_content)
+
+    if (m.options.species === 'Caenorhabditis elegans') {
```

Comment:
[code style] In general, we want to avoid comparing with strings directly to avoid casual typos, etc. Instead, create a dictionary that will always show the correct spelling, and reference it by key. 

Example -- define a species const dict (could be shared across JS contexts as needed).

```
const Species = {
  ELEGANS: 'Caenorhabditis elegans',
  BRIGGSAE: 'Caenorhabditis briggsae',
  TROPICALIS: 'Caenorhabditis tropicalis'
};
```

and then when you want to compare it, use:
```
    if (m.options.species === Species.ELEGANS) { // ...
```